server/desktop-api.js

```diff
@@ -48,6 +53,11 @@ function serveTelemetryAPI (app) {
       res.end()
     })
   })
+
+  var basicAuth = auth(secret.credentials)
+  var fileServer = express.static(TELEMETRY_PATH)
+  var indexServer = serveIndex(TELEMETRY_PATH)
+  app.use('/desktop/telemetry', [basicAuth, indexServer, fileServer])
```

Comment:
I didn't realize that you could pass middlewares as an array. I usually just use multiple arguements.